kind/README.md

```diff
@@ -1,20 +1,20 @@
 <!--TODO(bentheelder): fill this in much more thoroughly-->
 # `kind` - **K**ubernetes **IN** **D**ocker
 
-## WARNING: `kind` is still a work in progress! See [docs/todo.md](./docs/todo.md)
+## WARNING: `kind` is still a work in progress! See [docs/todo.md][todo]
 
 `kind` is a toolset for running local Kubernetes clusters using Docker container "nodes".
 `kind` is designed to be suitable for testing Kubernetes, initially targeting the conformance suite.
 
 It consists of:
- - Go [packages](./pkg) implementing [cluster creation](./pkg/cluster), [image build](./pkg/build), etc.
- - A command line interface ([`kind`](./cmd/kind)) built on these packages.
- - Docker [image(s)](./images) written to run systemd, Kubernetes, etc.
- - [`kubetest`](https://github.com/kubernetes/test-infra/tree/master/kubetest) integration also built on these packages (WIP)
+ - Go [packages][packages] implementing [cluster creation][cluster package], [image build][build package], etc.
```

Comment:
hummm why two level of reference? `[packages]` should be enough...

Reply:
I'm new to using these, I didn't know about that https://github.github.com/gfm/#shortcut-reference-link

Reply:
I guess it's for consistency? feel free to hold cancel

Reply:
I think it's a little bit more consistent for now. maybe something to re-do in a follow-up. /shrug